### PR TITLE
feat: ISIS brightness buttons push&hold

### LINKED ISSUE #4184: ISIS brightness buttons should be push&hold
**Mod Version**

{
  "built": "2021-03-28T16:25:26+00:00",
  "ref": "merge",
  "sha": "b280541409b0d46dd7513703640de179ccfb2126",
  "actor": "Duncan10",
  "event_name": "manual"
}

**Describe the bug**

The + and - brightness buttons on the ISIS need to be clicked repeatedly.
image

**To Reproduce**

1. Load up the plane
2. Change ISIS brightness
3.

**Expected behavior**

You should be able to click and hold the two buttons like you are able to with the MCDU brightness buttons

**Actual behavior**

**References**

IRL experience

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by 2hwk**
Fixes #4184

## Summary of Changes
Brightness buttons can now be held down to increase/decrease brightness (push and hold). As per issue, time taken to increase brightness to full from minimum is roughly 4 seconds.

## Screenshots (if necessary)

aBr61h3TRS

## References

See Issue #4184 

https://discord.com/channels/738864299392630914/758126266859388948/832179131152203806

## Additional context

Discord username (if different from GitHub): 2Cas#1022

## Testing instructions

1. In the self-test and altimeter views, use standby instrument's brightness push buttons to adjust the brightness of the display. 
2. Pressing and holding as well as single sharp presses should both work as expected.
3. Press the BUGS button. In this mode, the standby instrument's brightness push buttons should move the cursor that selects the heading/altitude bugs that are being adjusted by the altimeter knob. The brightness should not change. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml [hunk 1]
```diff
@@ -1099,7 +1099,6 @@
             </UseTemplate>
         </Component>
   </Template>
-
     <Template Name="A32NX_Push_BARO_Template">
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml:1102]: **question**: Is this template still used elsewhere?

Comment (2hwk) [reply flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml:1102]: Yes, it's used for the other push buttons on the standby indicator (no press and hold functionality for these for now)

**Comment by 2hwk** [comment]
> There is quite some code repetition. Is it possible to deduplicate the code?

Definitely. See new commit.

**Review by davidwalschots — APPROVED** [review]

**Comment by frankkopp** [comment]
Category: QA Tester
Name: Cdr_Maverick#6475
Date of testing: Current date 12/05/2021
Version of the sim: 1.15.10
PR Tested: #4184
Tier of testing: 1

Changes to observe:
1) In the self-test and altimeter views, use standby instrument's brightness push buttons to adjust
 the brightness of the display.
2) Pressing and holding as well as single sharp presses should both work as expected.
3) Press the BUGS button. In this mode, the standby instrument's brightness push buttons should move
 the cursor that selects the heading/altitude bugs that are being adjusted by the altimeter knob. The brightness should not change.

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Start C&D at gate
2. Powering up aircraft
3. Testing 1): during self test: short clicks and holding buttons work as expected
4. Testing 2): short clicks and holding buttons work as expected
5. Testing 3): After BUGS: short clicks move the cursor, no change in brightness. Holding the button has no additional effect

Issues (If Any): None

Overall Rating: Good
Conclusions of the testing: LGTM

**Comment by everythingaboutaviation** [comment]
Category: QA Trial
Name: everythingaboutaviation#9343
Date of testing: Current date 13/05/2021
Version of the sim:1.15.10.0
PR Tested: #4736
Tier of testing: 1

Changes to observe: 

1)In the self-test and altimeter views, use standby instrument's brightness push buttons to adjust
the brightness of the display.
2)Pressing and holding as well as single sharp presses should both work as expected.
3)Press the BUGS button. In this mode, the standby instrument's brightness push buttons should move
the cursor that selects the heading/altitude bugs that are being adjusted by the altimeter knob. The brightness should not change.

Testing technique: Following the instructions above.

Steps you've made to test the Pull Request:
1.Spawning at the gate
2.Setting up and powering up the A32NX
3.[Test 1] - Self test with short clicks and holding the buttons are working.
4.[Test 2] - Holding the buttons are working as expected.
5.[Test 3] - In BUGS mode, short clicks and holding the buttons wouldn't affect the brightness.

Overall Rating: Is good!

Conclusions of the testing: It worked.

**Comment by 2hwk** [comment]
There was an issue with the emissive code being overwritten by the XML template used, causing the button to be always emissive. Thankfully I have resolved this in a way that is actually simpler than the original PR, will open a new PR to fix.

**Pull request merged**